### ISSUE TITLE
Enhancement: SSD_SHEDULER env variable for start-stop-daemon

### ISSUE THREAD

**Issue by Forza-tng** [OpenRC/openrc#657]
I think we should have the env variable `SSD_SHEDULER`, similar to `SSD_IONICELEVEL` and `SSD_NICELEVEL`.

It is currently possible to use `start_stop_daemon_args="--scheduler xxxx`, but it would be more coherent to use the SSD_ style variable.

I'm currently involved with a new init.d script for a service that is very IO heavy, and letting users select `idle` or `batch` via conf.d would be great. Implementing the env variable would be much better than a service-specific option.

**Comment by FluffyFoxUwU** [comment]
(found this while Googling stuffs relating to this)

A little correction if its ok, don't you mean..
1. `SSD_IOSCHEDULER` and `--ionice` option if you meant I/O because `batch` is a process scheduler but you explained I/O
```
-I, --ionice <arg>                Set an ionice class:data when starting
```
2. `SSD_CPUSCHEDULER` and `--scheduler` option because `batch` is one of CPU/process schedulers
```
      --scheduler <arg>             Set process scheduler
```

Note: Help information is from start-stop-daemon OpenRC version 0.53

**Comment by Forza-tng** [comment]
Thanks for your comments. I meant, as you guessed, CPU scheduler and not IO scheduler. It is possible to use `--scheduler`, but I do not think there is a [corresponding ENV variable](https://github.com/OpenRC/openrc/blob/c45fe9fba5f9149d17eab9e1bfef741a633b1646/man/start-stop-daemon.8#L197) for it. `SSD_CPUSCHEDULER` would be a good name.

**Comment by FoxieFlakey** [comment]
Yes and I suggest that more specific `SSD_CPUSCHEDULER_PRIO` override the `SSD_NICELEVEL` or warn the user that both is defined in environment as niceness level is used to calculate priority by the kernel.

**Comment by Forza-tng** [comment]
If I'm not mistaken, `SCHED_OTHER/NORMAL`, `SCHED_BATCH` and `SCHED_IDLE` uses niceness and ignores priority, while `SCHED_FIFO` and `SCHED_RR` uses priority and ignores niceness.

While SSD could make users aware of this by printing a warning, little else is gained by hard coding checks. What if the kernel adds or changes schedulers and we have hardcoded these options?

https://www.kernel.org/doc/html/latest/scheduler/sched-design-CFS.html
